students/JaeKim/lesson09/assignment/lesson02/assignment/charges_calc.py

```diff
@@ -0,0 +1,111 @@
+'''
+Returns total price paid for individual rentals 
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+import sys
+
+def disable_logging(func):
+    def logged(*args, **kwargs):
+        logging.disabled = True
+        result = func(*args, **kwargs)
+        logging.disabled = False
+        return result
+    return logged
+
+def parse_cmd_arguments():
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='ouput JSON file', required=True)
+    parser.add_argument('-d', '--debug', help='debug level', required=False, default=0)
+    return parser.parse_args()
```

Comment:
The implementation through args is difficult. Fortunately, we rarely have to use args to control a decorator. :)